feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableItemDetailSummaryCard.kt

```diff
@@ -36,11 +35,7 @@ fun TimetableItemDetailSummaryCard(
     Card(
         shape = RoundedCornerShape(12.dp),
         modifier = modifier,
-        colors = CardDefaults.cardColors(
-            containerColor = MaterialTheme.colorScheme.surfaceColorAtElevation(
-                1.dp,
-            ),
-        ),
+        colors = CardDefaults.cardColors(containerColor = Color(0xFFF2F4F1)),
```

Comment:
We might have to use theme color for applying a dark theme and it seems like we can apply it by adding elevation. 

<img width="226" alt="image" src="https://github.com/DroidKaigi/conference-app-2023/assets/1386930/c8592941-4b3b-48b9-bad9-7f9af8b9de46">


> Surface at elevation +1 becomes Surface Container Low
https://material.io/blog/tone-based-surface-color-m3

The problem is how we can get elevation +1 😓 

https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/ColorScheme.kt;l=478-484;drc=033f1c4e0e01fb2702b8f59520777aadaed46e5a

Reply:
You can leave this color as it is `surfaceColorAtElevation(1.dp,)` in this PR. In that case please create the issue for this 🙏

Reply:
I think it would be better to leave the color as `surfaceColorAtElevation(1.dp,)`. I modified it at https://github.com/DroidKaigi/conference-app-2023/pull/508/commits/9db725f8719ea0e9c52d1bb3fe5e8adb692562dc . 

Also, I made the following issue for the card background color issue. ([link](https://github.com/DroidKaigi/conference-app-2023/issues/514))